### DISCUSSION TITLE
How to balance the brightness of all monitors to have a consistent experience?

### DISCUSSION THREAD

**Discussion by yuchen-xjtu** [MonitorControl/MonitorControl#1558] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

For my MacBook has a 500nit max brightness, and my external monitor has a 400nit max brightness, To set them in the same brightness percentage caused internal and external display having different sensual brightness almost all the time.

### Describe the solution you'd like

you can link a certain percentage of internal and external display. for example, you can link the 80% brightness of internal display with 95% brightness of external display; and  link the 60% brightness of internal display with 75% brightness of external display. so my internal display can has a fixed percentage brightness lower than my external display to have a consistent experience.

### Describe alternatives you've considered

and they can also both reach 100% of brightness if you need.

**Comment by waydabber** [comment]
Sadly this is not possible with MonitorControl. I plan to add a calibration feature to [BetterDisplay](https://github.com/waydabber/BetterDisplay) to equalize the brightness of synced displays. I also recommend https://lunar.fyi by @alin23 as it has this feature already and a self-learning algorithm to figure out how displays with various brightnesses should work together.
